bears/uploadTool.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import glob
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+# inserting path here so the following imports work wherever you run the script
+# in coala-bears
+sys.path.insert(0, '..')
```

Comment:
I feel like this tool doesn't belong into this repository, also this path modification doesn't make any difference since you're not importing anything from bears